tools/gluten-te/dockerfile-buildenv-ubuntu22.04

```diff
@@ -0,0 +1,67 @@
+FROM ubuntu:22.04 AS gluten-buildenv
```

Comment:
Is content of the file same with 20.04? If yes using an arg maybe help to avoid duplicating codes

Like this example https://www.jeffgeerling.com/blog/2017/use-arg-dockerfile-dynamic-image-specification

Reply:
Thanks, I will follow it.